pkg/api/meta/finalizers.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// GetFinalizers gets the list of finalizers on obj
```

Comment:
You can obviously check for existance of a finalizer with this call and spinning the list yourself, just curious (especially if upstreaming and we're speaking of convenience functions) if there should be HasFinalizer(obj, value) that returns bool if there's a finalizer with the given value.